models/yang/sonic/sonic-port.yang

```diff
@@ -56,7 +61,9 @@ module sonic-port {
 				}
 
 				leaf description {
-					type string;
+					type string {
+					    pattern '[ -~]{0,64}';
```

Comment:
Did not understand the regular expression. '^' is missing if we want to exclude ' -~' in description.

Reply:
Ok, got it. Its is all characters between '<space> ' and '~'.